packages/ui-checkbox/src/Checkbox/props.ts

```diff
@@ -40,17 +40,39 @@ type CheckboxOwnProps = {
   label: React.ReactNode
   id?: string
   value?: string | number
+  /**
+   * object with shape: `{
+   * text: PropTypes.node,
+   * type: PropTypes.oneOf(['error', 'hint', 'success', 'screenreader-only'])
+   *   }`
+   */
```

Comment:
```suggestion
  /**
   * Array of objects with shape: `{
   *   text: ReactNode,
   *   type: One of: ['error', 'hint', 'success', 'screenreader-only']
   *   }`
   */
```

(this is the format we use in other input components too)